web-wallet/src/routes/(app)/dashboard/receive/+page.svelte

```diff
@@ -0,0 +1,30 @@
+<svelte:options immutable={true} />
+
+<script>
+  import { onDestroy, onMount } from "svelte";
+  import { TransferContract } from "$lib/containers";
+  import { IconHeadingCard } from "$lib/containers/Cards";
+  import { contractDescriptors, updateOperation } from "$lib/contracts";
+  import { operationsStore } from "$lib/stores";
+  import { mdiArrowTopRight } from "@mdi/js";
+
+  onMount(() => {
+    updateOperation("receive");
+  })
+
+  onDestroy(() => {
+    updateOperation("receive");
+  });
+</script>
+
+<IconHeadingCard
+  gap="medium"
+  heading="Transfer"
+  iconPath={mdiArrowTopRight}
+  reverse
+>
+  <TransferContract
+    descriptor={contractDescriptors[1]}
```

Comment:
We should have a function to retrieve them by ID.

Reply:
I think we should remove the contract descriptors.